OJP/OJP_Trips.xsd

```diff
@@ -943,13 +940,16 @@
 					<xs:documentation>If there are more than 2 lanes, then TurnAction half_right,right,sharp_right may help decide</xs:documentation>
 				</xs:annotation>
 			</xs:enumeration>
-			<!-- undefined values -->
 			<xs:enumeration value="unknown_turning_kind"/>
+			<!-- additional enum values for individual vehicles -->
```

Comment:
Did you move those comments on purpose?

Reply:
no, I think this is something that the post processing in the script does. It doesn't like <!-- --> style comments within xs:enumeration. Perhaps we should remove them.